apps/server/src/modules/tldraw/repo/tldraw-board.repo.spec.ts

```diff
@@ -0,0 +1,167 @@
+import { Test } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import WebSocket from 'ws';
+import { WsAdapter } from '@nestjs/platform-ws';
+import { CoreModule } from '@src/core';
+import { ConfigModule } from '@nestjs/config';
+import { createConfigModuleOptions } from '@src/config';
+import { config } from '@src/modules/tldraw/config';
+import { TldrawBoardRepo } from '@src/modules/tldraw/repo/tldraw-board.repo';
```

Comment:
same like in other places use relative pathes inside the module. I do not write in on each file, please check all.